### PR TITLE
added first version of map

### LINKED ISSUE #55: Map tile
We will soon have the map of the expo. Create a tile that spans the whole width of the front page container and shows the map and the legend for it.
Similar to [last year](https://2023.konferencia.simonyi.bme.hu/) 
Code: https://github.com/simonyiszk/konf-web-2023/blob/main/src/components/map/Floorplan.tsx
Would be cool if clicking on the companies would highlight them on the map, just like last year

not so final version of the expo map: https://drive.google.com/file/d/1PII4yUuTx-1TYUk5LjM14pzb4UAtKw_S/view?usp=drive_link

### PR DESCRIPTION
**Pull request by FearsomeRover**
I added the first iteration of the map tile. I dont really like the colors on it, but take a look pls, an decide, what we should change. I also think the pulsing animation is hard to notice, so we could maybe replace that. The problem is, that for scaling up the rectangles we should first need to get rid of the transform, as it messes the scaling up in the animation.

### CODE REVIEW

**Comment by FearsomeRover** [comment]
Ohh, i also removed the scroll effect from last year, as it was kinda glitching for me.

File: src/components/tiles/map-tile/floorplan.tsx [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+  'Nova',
+  'KUKA',
+  'OTP',
+  'Knorr-Bremse',
+  'SCHIS',
+  'Paripa',
+  'BME Suborbitals',
+  'BME Formula Racing Team',
+  'Schdesign',
+  'Kir-dev',
```
Comment (Tschonti) [inline src/components/tiles/map-tile/floorplan.tsx]: schdesign
Kir-Dev

I know, the PDF I sen you was also incorrect, but that's not public at least :D

File: src/components/tiles/map-tile/floorplan.tsx [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+  'SEM',
+  'HA5KFU',
+  'LEGO',
+];
+
+export function Floorplan() {
+  const [active, setActive] = useState(0);
```
Comment (Tschonti) [inline src/components/tiles/map-tile/floorplan.tsx]: I think it's better if nothing is active by default

File: src/components/tiles/map-tile/floorplan.tsx [hunk 3]
```diff
@@ -0,0 +1,76 @@
...
+  return (
+    <section className='container'>
+      <h4 className='mb-4 text-center text-4xl font-black md:text-6xl lg:col-span-2 lg:text-8xl'>Térkép</h4>
```
Comment (Tschonti) [inline src/components/tiles/map-tile/floorplan.tsx]: It's bigger than any other text on the website, yet it's a h4

I think just use an h2 like on the other tiles

File: src/components/tiles/map-tile/floorplan.tsx [hunk 4]
```diff
@@ -0,0 +1,76 @@
...
+      <div className='relative flex flex-col-reverse md:flex-row w-full rounded-lg justify-between gap-4 items-center'>
```
Comment (Tschonti) [inline src/components/tiles/map-tile/floorplan.tsx]: I think I'd prefer justify center with some gap in the middle

File: src/components/tiles/map-tile/floorplan.tsx [hunk 5]
```diff
@@ -0,0 +1,76 @@
...
+        <ol className='pl-3 text-center grid grid-cols-2 gap-4 md:block'>
```
Comment (Tschonti) [inline src/components/tiles/map-tile/floorplan.tsx]: text-center is a bit weird imo

**Review by Tschonti — COMMENTED** [review]
I proposed a few styling changes in the comments, but you don't have to agree xd

The scrolling would be great at least on mobile, since the map and the list can't be seen side-by-side.

Also, the list is weird on mobile:
image

File: src/components/tiles/map-tile/floorplan.tsx (hunk 2 above)
Comment (triszt4n) [reply src/components/tiles/map-tile/floorplan.tsx]: my sugg: `= useState<number>(-1)`

File: src/components/tiles/map-tile/floorplan.tsx [hunk 6]
```diff
@@ -0,0 +1,76 @@
...
+  'Regisztrációs pult',
+  'Bosch',
+  'Nokia',
+  'ICF Tech',
+  'Sicontact',
+  'Silicon Labs',
+  'Interactive Brokers',
+  'Paks II.',
+  'Barré',
+  'Nova',
```
Comment (triszt4n) [inline src/components/tiles/map-tile/floorplan.tsx]:
```suggestion
  'Nova Services',
```

File: src/components/tiles/map-tile/floorplan.tsx [hunk 7]
```diff
@@ -0,0 +1,76 @@
...
+  'KUKA',
+  'OTP',
```
Comment (triszt4n) [inline src/components/tiles/map-tile/floorplan.tsx]:
```suggestion
  'OTP Bank',
```

File: src/components/tiles/map-tile/floorplan.tsx [hunk 8]
```diff
@@ -0,0 +1,76 @@
...
+  'Knorr-Bremse',
+  'SCHIS',
+  'Paripa',
```
Comment (triszt4n) [inline src/components/tiles/map-tile/floorplan.tsx]:
```suggestion
  'PARIPA',
```

File: src/components/tiles/map-tile/floorplan.tsx [hunk 9]
```diff
@@ -0,0 +1,76 @@
...
+  'BME Suborbitals',
+  'BME Formula Racing Team',
+  'Schdesign',
```
Comment (triszt4n) [inline src/components/tiles/map-tile/floorplan.tsx]:
```suggestion
  'schdesign',
```

File: src/components/tiles/map-tile/floorplan.tsx [hunk 10]
```diff
@@ -0,0 +1,76 @@
...
+  'Kir-dev',
+  'SEM',
+  'HA5KFU',
+  'LEGO',
```
Comment (triszt4n) [inline src/components/tiles/map-tile/floorplan.tsx]:
```suggestion
  'LEGO Kör',
```

**Review by triszt4n — CHANGES_REQUESTED** [review]
sry for bad PDF from before, added suggs for the exact names

File: src/components/tiles/map-tile/floorplan.tsx (hunk 5 above)
Comment (FearsomeRover) [reply src/components/tiles/map-tile/floorplan.tsx]: That was meant to be only on mobile...

**Comment by FearsomeRover** [comment]
Well on mobile it was weird, how far down the end of the text is from the map. But I changed the texts, and layout on mobile.

File: src/components/tiles/map-tile/floorplan.tsx [hunk 11]
```diff
@@ -0,0 +1,76 @@
...
+  'Sicontact',
+  'Silicon Labs',
+  'Interactive Brokers',
+  'Paks II.',
+  'Barré',
+  'Nova Services',
+  'KUKA',
+  'OTP Bank',
+  'Knorr-Bremse',
+  'SCHIS',
```
Comment (Tschonti) [inline src/components/tiles/map-tile/floorplan.tsx]: Schönherz Iskolaszövetkezet, right @triszt4n ?

**Comment by Tschonti** [comment]
I re-enabled the scroll on screens where the map and the list is not side-by-side, what do you think @FearsomeRover?

**Comment by FearsomeRover** [comment]
mdx hmmmmmmmm. 
I think the scrolling is great on small screens. The yellow color is still not attention grabbing for me. Tomorrow i will see, if i can do some more flashy growing or pulsing effects. When are we supposed to get this into prod?
"3. fázis (márc. 2. – )" is not very informative about this.

**Comment by Tschonti** [comment]
The map was hard to read, so I wanted to use the flex-col on slightly larger screens than md, but it was weird with lg. mdx is a custom breakpoint between the two (<940px) I defined in `tailwind.config.ts` for something else a few weeks ago.

I think the yellow flashing is alright, but feel free to experiment.

According to the log of today's meeting, we should release today. But we can always improve the animation later

**Review by Tschonti — APPROVED** [review]
great work, thx for the help!

**Pull request merged**